cluster/src/main/scala/io/snappydata/cluster/ExecutorInitiator.scala

```diff
@@ -32,6 +29,8 @@ import com.gemstone.gemfire.distributed.internal.membership.InternalDistributedM
 import com.gemstone.gemfire.internal.cache.GemFireCacheImpl
 import com.pivotal.gemfirexd.internal.engine.Misc
 import com.pivotal.gemfirexd.internal.engine.distributed.utils.GemFireXDUtils
+import com.pivotal.gemfirexd.internal.engine.store.ServerGroupUtils
+import io.snappydata.gemxd.ClusterCallbacksImpl
```

Comment:
Not unused, it is showing changes in this file because I had earlier made changes to it which were removed later.